todo/src/App.js

```diff
@@ -1,27 +1,54 @@
-// import logo from './logo.svg';
-import {useState} from 'react';
-
-import logo from './tek_logo.png';
+import React, {useState} from 'react';
 import './App.css';
 import Todo from './Components/Todo';
+import TodoForm from './Components/TodoForm';
+
 
```

Comment:
For the Design, I would say the 'Enter' button is missing. Otherwise, people need to make an assumption.

Reply:
The complete button can't be unclicked.